src/go/rpk/pkg/cli/cmd/redpanda/admin/brokers/decommission.go

```diff
@@ -47,16 +47,19 @@ cluster is unreachable), use the hidden --force flag.
 
 			if !force {
 				brokers, err := cl.Brokers(cmd.Context())
-				out.MaybeDie(err, "unable to get broker list: %v; to bypass the node version check re-run this with --force; see this command's help text for for more details", err)
+				out.MaybeDie(err, "unable to get broker list: %v; to bypass the node version check re-run this with --force; see this command's help text for more details", err)
 
 				var (
 					b             admin.Broker
 					found, anyOld bool
 				)
 				for _, br := range brokers {
 					if br.NodeID == broker {
+						if br.Version == "" {
+							out.Exit("version for broker %d is unknown. The node is likely dead?\nto bypass the node version check re-run this with --force; see this command's help text for more details", br.NodeID)
```

Comment:
Can this change to 
```
version for broker %d is unknown, is node the node offline?\nto bypass the node version check re-run this with --force; see this command's help text for more details
```

Reply:
Sounds good, thanks. 
Just wanted to understand the implication; shouldn't we use the word 'dead' or something else?

Reply:
I suppose we could, that works for me too. But, what does dead mean? Is offline not accurate? U

Reply:
Thanks for making it clear. Indeed 'offline' is more accurate than 'dead' as it sounds only applicable to unexpected node down. 

Will go with 'offline'!